docs/getting-started/configuring-resoto.md

```diff
@@ -364,3 +364,9 @@ gcp:
 ```
 
 The setting `resotoworker.pool_size` determines how many collectors (aws, gcp, digitalocean, k8s, etc.) are run concurrently. `aws.account_pool_size` and `gcp.project_pool_size` are used to determine how many accounts or projects respectively are collected concurrently. Within AWS the setting `aws.region_pool_size` is used to determine how many regions per account are collected concurrently.
+
+## Configuring logging
```

Comment:
```suggestion
## Logging
```

Reply:
idk, the list so far is
[Listing Configurations](https://resoto.com/docs/getting-started/configuring-resoto#listing-configurations)
[Editing Configuration](https://resoto.com/docs/getting-started/configuring-resoto#editing-configuration)
[Setting Individual Properties](https://resoto.com/docs/getting-started/configuring-resoto#setting-individual-properties)
[Overriding Individual Properties](https://resoto.com/docs/getting-started/configuring-resoto#overriding-individual-properties)
[Restoring the Default Configuration](https://resoto.com/docs/getting-started/configuring-resoto#restoring-the-default-configuration)
[Configuring Cloud Providers](https://resoto.com/docs/getting-started/configuring-resoto#configuring-cloud-providers)
[Configuring Resoto Worker for many-core machines](https://resoto.com/docs/getting-started/configuring-resoto#configuring-resoto-worker-for-many-core-machines)

Wouldn't it look odd if the last entry is just `Logging`?

Reply:
Hm, I guess let's leave it for now but I think that the headings should be "context-aware," if that makes sense.  I'll give it some thought and edit it later.

It should be title case though, "Configuring Logging."